front/src/pages/AddItem/AddItem.tsx

```diff
@@ -1,28 +1,48 @@
-import React from 'react';
+import React, { useRef } from 'react';
 import styles from './AddItem.module.scss';
+import useAddTodo from '../../hooks/useAddTodo';
+import {
+  TodoImportance,
+  TodoImportanceValues,
+  TodoToCreate,
+} from '../../types';
+import { capitalize } from '../../helpers';
 
 const AddItem = () => {
+  const { mutate: addTodo } = useAddTodo(resetForm);
```

Comment:
I don't like this passing `resetForm` as an argument. This way `useAddTodo` not just handle async operation but also takes callbacks to call inside.

In my opinion, having `mutate(newToDo).then(resetForm)` or rather `addTodo(newToDo).then(resetForm)` is more flexible. And follows single responsibility principle for `useAddTodo` hook

Reply:
@skyboyer 
fixed.